cmd/antrea-agent/agent.go

```diff
@@ -137,7 +137,7 @@ func run(o *Options) error {
 
 	enableAntreaIPAM := features.DefaultFeatureGate.Enabled(features.AntreaIPAM)
 	enableBridgingMode := enableAntreaIPAM && o.config.EnableBridgingMode
-	enableNodePortLocal := features.DefaultFeatureGate.Enabled(features.NodePortLocal) && o.config.NodePortLocal.Enable
+	enableNodePortLocal := o.config.NodePortLocal.Enable
```

Comment:
Please check this comment for backward compatibility.
https://github.com/antrea-io/antrea/pull/5401/files#r1308177855

Reply:
Added a check for feature gates to ensure compatibility.